src/lib/mollie/api/resource/base.coffee

```diff
@@ -51,31 +51,39 @@ module.exports = class Base
 				if (body.error)
 					return callback(body);
 				callback(this.copy(body, new this.constructor.object));
-		);
+		).on("error", (error) => 
```

Comment:
If you do nothing else but returning the error object. There is no need to provide the request object to the resource base class. You can simply call the callback with the error object in the client. Right now you are just introducing a lot of duplicated code in this file.